website/docs/r/emr_cluster.html.markdown

```diff
@@ -282,7 +284,8 @@ Supported arguments for the `core_instance_group` configuration block:
 
 * `instance_type` - (Required) EC2 instance type for all instances in the instance group.
 * `autoscaling_policy` - (Optional) String containing the [EMR Auto Scaling Policy](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-automatic-scaling.html) JSON.
-* `bid_price` - (Optional) Bid price for each EC2 instance in the instance group, expressed in USD. By setting this attribute, the instance group is being declared as a Spot Instance, and will implicitly create a Spot request. Leave this blank to use On-Demand Instances.
+* `market` - (Required) The instance purchasing option. Valid values are ON_DEMAND or SPOT. 
```

Comment:
Once this is made optional, we should alphabetize it with the other optional arguments and also explain its backwards compatible behavior:

```suggestion
* `market` - (Optional) The instance purchasing option. Valid values are `ON_DEMAND` or `SPOT`. Defaults to `ON_DEMAND` unless `bid_price` is set to a non-empty value.
```

Similar changes to `master_instance_group` below.